blocks/init/src/Blocks/manifest.json

```
@@ -60,27 +60,27 @@
 			{
 				"name": "Ocean",
 				"slug": "ocean",
-				"color": "#44AEDE"
+				"color": "#155D7E"
```

```suggestion
				"color": "#00B8D4"
```
Needs to have black text for WCAG AAA.